src/analysis/liveness-transfer-function.h

```diff
@@ -0,0 +1,101 @@
+#ifndef wasm_analysis_liveness_transfer_function_h
+#define wasm_analysis_liveness_transfer_function_h
+
+#include "lattice.h"
+#include "monotone-analyzer.h"
+
+namespace wasm::analysis {
+
+class LivenessTransferFunction : public Visitor<LivenessTransferFunction> {
+  FinitePowersetLattice::Element* currState;
```

Comment:
```suggestion
  FinitePowersetLattice::Element* currState = nullptr;
```
This is a little more idiomatic than doing `currState(nullptr)` in modern C++, I think. It also scales better with multiple constructors (which this doesn't have yet, but might some day).